src/main/kotlin/org/cascadebot/cascadebot/commands/moderation/UnlockCommand.kt

```diff
@@ -0,0 +1,73 @@
+package org.cascadebot.cascadebot.commands.moderation
+
+import javassist.NotFoundException
+import net.dv8tion.jda.api.Permission
+import net.dv8tion.jda.api.entities.ISnowflake
+import net.dv8tion.jda.api.entities.Member
+import net.dv8tion.jda.api.entities.Role
+import net.dv8tion.jda.api.entities.TextChannel
+import net.dv8tion.jda.api.exceptions.PermissionException
+import org.cascadebot.cascadebot.commandmeta.CommandContext
+import org.cascadebot.cascadebot.commandmeta.MainCommand
+import org.cascadebot.cascadebot.commandmeta.Module
+import org.cascadebot.cascadebot.data.managers.LockManager
+import org.cascadebot.cascadebot.permissions.CascadePermission
+import org.cascadebot.cascadebot.utils.DiscordUtils
+
+class UnlockCommand : MainCommand() {
```

Comment:
We might want to do a thing where we store when they locked a channel and if it's been a while then we warn the user that it will reset permissions to before the channel was lock, either that or we find a way to handle permissions being changed while a channel is locked.

Reply:
I don't think we need to do this since our lock command only adds 1 permission override per "lock"

Reply:
Yegh my bad, for some reason I thought it was storing all permissions before the back, but apon further investigation it's not.